### PR TITLE
CORE-1521 "data storage" in features.md

### PR DESCRIPTION
**Pull request by dzajkowski**
## Overview
added scenario for data handling

### JIRA ticket:
<sup>CORE-1521</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Comment by dzajkowski** [comment]
bors try

**Comment by dzajkowski** [comment]
bors try

**Comment by dzajkowski** [comment]
bors try

File: integration-tests/features/contracts/storage/read-data.rho [hunk 1]
```diff
@@ -0,0 +1,5 @@
+new stdout(`rho:io:stdout`) in {
+  for(v <- channel1) {
```
Comment (goral09) [inline integration-tests/features/contracts/storage/read-data.rho]: You have to define what is `channel1`.

**Pull request merged**